src/store/modules/swap/ethBancor.ts

```diff
@@ -3388,10 +3378,26 @@ export class EthBancorModule
           maxStakes.maxAllowedTknWei,
           tknReserve.decimals
         )
-      },
-      "asaf"
+      }
+    }
+  }
+
+  @action async calculateProtectionSingle({
+    poolId,
+    reserveAmount
+  }: {
+    poolId: string;
+    reserveAmount: ViewAmount;
+  }): Promise<ProtectionRes> {
+    const depositingNetworkToken = compareString(
+      this.liquidityProtectionSettings.networkToken,
+      reserveAmount.id
     );
 
+    const inputToken = this.token(reserveAmount.id);
+    
+    const { maxStakes } = await this.getMaxStakes({poolId})
```

Comment:
@pingustar @H34D is someones prettier config off? ```({poolId})``` should have got auto-formatted to ```({ poolId })```